course_catalog/settings.py

```diff
@@ -120,13 +120,20 @@
 )
 DEFAULT_DATABASE_CONFIG['CONN_MAX_AGE'] = get_int('COURSE_CATALOG_DB_CONN_MAX_AGE', 0)
 
-if get_bool('COURSE_CATALOG_DB_DISABLE_SSL', False):
+if get_bool('COURSE_CATALOG_DB_DISABLE_SSL', True):
     DEFAULT_DATABASE_CONFIG['OPTIONS'] = {}
 else:
     DEFAULT_DATABASE_CONFIG['OPTIONS'] = {'sslmode': 'require'}
 
 DATABASES = {
-    'default': DEFAULT_DATABASE_CONFIG
+    'default': {
```

Comment:
Keep as is (`DEFAULT_DATABASE_CONFIG`), app should be run via `docker-compose up`